packages-content-model/roosterjs-content-model-dom/lib/domUtils/entityUtils.ts

```diff
@@ -61,14 +61,16 @@ export function generateEntityClassNames(format: ContentModelEntityFormat): stri
 }
 
 /**
- * @internal
+ * Checks whether the node provided is a Entity delimiter
+ * @param node the node to check
+ * @return true if it is a delimiter
  */
-export function isEntityDelimiter(element: HTMLElement): boolean {
+export function isEntityDelimiter(node: Node): node is HTMLElement {
```

Comment:
minor: When return "node is HTMLElement", it has a side effect.

For example:
```ts
const node = .... (a html element but is not a delimiter)

if (isEntityDelimiter(node)) {
 ... //ok
} else {
  // Here, TypeScript will treat as a Node but it cannot be an HTMLElement since isEntityDelimiter() returns false which means "node is not HTMLElement"
}
```

Reply:
Hmm I see, reverted this to original code